api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -236,6 +256,9 @@ message AccessLogCommon {
   // For HTTP: Total number of bytes received from the upstream by the http stream.
   // For TCP: Total number of bytes sent to the upstream by the tcp proxy.
   uint64 upstream_wire_bytes_received = 32;
+
+  // The type of the access log, which indicates when the log was recorded.
+  AccessLogType access_log_type = 33;
```

Comment:
Looking at the comments for `intermediate_log_entry`, there are additional constraints/considerations:
```
  // There may be multiple intermediate log entries and only one final log entry for each
  // long-live stream (TCP connection, long-live HTTP2 stream).
  // And if it is necessary, unique ID or identifier can be added to the log entry
  // :ref:`stream_id <envoy_v3_api_field_data.accesslog.v3.AccessLogCommon.stream_id>` to
  // correlate all these intermediate log entries and final log entry.
```

Are there any constraints for the enum-values? Are there additional considerations?
If so, please update this field's comments accordingly.